src/libraries/System.Console/ref/System.Console.cs

```diff
@@ -8,58 +8,81 @@ namespace System
+        [System.Runtime.Versioning.UnsupportedOSPlatformAttribute("browser")]
         public static int CursorTop { get { throw null; } set { } }
+        [System.Runtime.Versioning.UnsupportedOSPlatformAttribute("browser")]
         public static bool CursorVisible { [System.Runtime.Versioning.SupportedOSPlatformAttribute("windows")] get { throw null; } set { } }
         public static System.IO.TextWriter Error { get { throw null; } }
+        [System.Runtime.Versioning.UnsupportedOSPlatformAttribute("browser")]
         public static System.ConsoleColor ForegroundColor { get { throw null; } set { } }
-        public static System.IO.TextReader In { get { throw null; } }
-        public static System.Text.Encoding InputEncoding { get { throw null; } set { } }
+        public static System.IO.TextReader In { [System.Runtime.Versioning.UnsupportedOSPlatformAttribute("browser")] get { throw null; } }
+        public static System.Text.Encoding InputEncoding { [System.Runtime.Versioning.UnsupportedOSPlatformAttribute("browser")] get { throw null; } set { } }
         public static bool IsErrorRedirected { get { throw null; } }
         public static bool IsInputRedirected { get { throw null; } }
         public static bool IsOutputRedirected { get { throw null; } }
         public static bool KeyAvailable { get { throw null; } }
-        public static int LargestWindowHeight { get { throw null; } }
-        public static int LargestWindowWidth { get { throw null; } }
+        public static int LargestWindowHeight { [System.Runtime.Versioning.UnsupportedOSPlatformAttribute("browser")] get { throw null; } }
+        public static int LargestWindowWidth { [System.Runtime.Versioning.UnsupportedOSPlatformAttribute("browser")] get { throw null; } }
         [System.Runtime.Versioning.SupportedOSPlatformAttribute("windows")]
         public static bool NumberLock { get { throw null; } }
         public static System.IO.TextWriter Out { get { throw null; } }
         public static System.Text.Encoding OutputEncoding { get { throw null; } set { } }
+        [System.Runtime.Versioning.UnsupportedOSPlatformAttribute("browser")]
         public static string Title { [System.Runtime.Versioning.SupportedOSPlatformAttribute("windows")] get { throw null; } set { } }
+        [System.Runtime.Versioning.UnsupportedOSPlatformAttribute("browser")]
         public static bool TreatControlCAsInput { get { throw null; } set { } }
+        [System.Runtime.Versioning.UnsupportedOSPlatformAttribute("browser")]
         public static int WindowHeight { get { throw null; } [System.Runtime.Versioning.SupportedOSPlatformAttribute("windows")] set { } }
         public static int WindowLeft { get { throw null; } [System.Runtime.Versioning.SupportedOSPlatformAttribute("windows")] set { } }
         public static int WindowTop { get { throw null; } [System.Runtime.Versioning.SupportedOSPlatformAttribute("windows")] set { } }
```

Comment:
Are these supported?

Reply:
From the list in `pnse-17082020.txt` under https://github.com/dotnet/runtime/issues/41087, both `set_WindowLeft` and `set_WindowTop` are to be marked as unsupported on Browser. However because it was marked as `SupportedOSPlatformAttribute("windows")` I assumed that meant the API is only supported on windows and unsupported on all other OS platforms, so I opted to exclude `UnsupportedOSPlatformAttribute("browser")`.

After looking into the attributes more and finding this https://github.com/terrajobst/designs/blob/master/accepted/2020/platform-exclusion/platform-exclusion.md, I think that assumption was correct and marking both `SupportedOSPlatformAttribute` and `UnsupportedOSPlatformAttribute` for different platforms both as their lowest levels would be considered an inconsistent list.

Reply:
I see